src/main/java/vct/col/rewrite/JavaEncoder.java

```diff
@@ -255,15 +260,18 @@ public void visit(ASTClass cl){
             String internal_name=create_method_name(INTERNAL, cl_type ,m);
             boolean varArgs=m.usesVarArgs();
             res.add(create.method_kind(m.kind, returns, external_contract, external_name, parameters, varArgs, null));
-            BlockStatement body=create.block();
-            body.add(create.comment("//TODO: unfolds of chained predicates in pre-condition"));
-            body.add(create.invokation(
-                create.reserved_name(ASTReserved.Super),
-                null,
-                create_method_name("internal", cl.super_classes[0],m),
-                get_names(parameters)));
-            body.add(create.comment("//TODO: folds of chained predicates in post-condition"));
-            res.add(create.method_kind(m.kind, returns, internal_contract, internal_name, parameters, varArgs, body));
+            // We leave body empty, as the method is guaranteed to have no contract
+            // Therefore no extra proof steps (unfolding predicates, checking pre/post conditions for implications)
+            // are not necessary and we can leave the method abstract
+//            BlockStatement body=create.block();
```

Comment:
Should we remove this code instead of commenting it?

Reply:
There is also a sonar cloud recommendation on this file: "Use already-defined constant 'INTERNAL' instead of duplicating its value here." Could you fix that now that you are modifying the file? :D

Reply:
Fixed in 345a7eb.